src/raster/r.boxplot/r.boxplot.html

```diff
@@ -0,0 +1,107 @@
+<h2>DESCRIPTION</h2>
+
+<p>
+<em>r.boxplot</em> draws boxplots of the raster values of a input raster
+layer. The user has the option to define a zonal (categorical) layer. In that
+case, a boxplot will be drawn for each zone, using the values of the input
+raster that fall within that zone.
+
+<p>
+By default, the resulting plot is displayed on screen (default). 
+However, the user can also save the plot to file using the 
+<em>output</em> option. The format is determined by the extension 
+given by the user. So, if output = outputfile.png, the plot will 
+be saved as a png file.
+
+<p>
+The whiskers extend to the most extreme data point, which is no more than
+<i>range</i> &#10005; the interquartile range (iqr) from the box. By default,
+a <i>range</i> of 1.5 is used, but the user can change this. Note that range
+values need to be larger than 0.
+
+<p>
+By default, outliers are not included in the plot. Set to -o flag to include
+them in the plot. To create a point layer with the locations of the outliers, 
+the user needs to provide the name of the output map using <i>map_outliers</i>.
+
+<p>
+There are a few layout options, including the option to rotate 
+the plot and the x-axis labels, print the boxplot(s) with notches and
+sort the boxplot from low to high (ascending) or from high to low (descending)
+median.
+
+<p>
+If the user opts to have the outliers drawn, a vector layer will be created
+with the locations of the outliers. 
```

Comment:
IIUC, this is different than what is said above, where it is stated that to get the vector layer, the user needs to set the `map_outliers` option